### ISSUE TITLE
Add Bodycare, Ramsdens, Klass, Neon Sheep and Greenhalgh's 🇬🇧

### ISSUE THREAD

**Issue by lakedistrictOSM** [osmlab/name-suggestion-index#4290]
locationSet ["gb"] for all of the below:

- [x] brand=Ramsdens
       brand:wikidata=Q99938502
       name=Ramsdens
       shop=pawnbroker

- [x] brand=Klass
       brand:wikidata=Q99938733
       name=Klass
       shop=clothes
- [x] brand=Greenhalgh's
       brand:wikidata=Q99939079
       name=Greenhalgh's
       shop=bakery
- [x] brand=Bodycare
       brand:wikidata=Q99938329
       name=Bodycare
       shop=chemist

- [x] brand=Neon Sheep
       brand:wikidata=Q99938913
       name=Neon Sheep
       shop=gift

**Comment by UKChris-osm** [comment]
Added Ramsdens via [5f50e32](https://github.com/osmlab/name-suggestion-index/commit/5f50e32f6dc74334fb408cb69a76c86b201370b5)

**Comment by UKChris-osm** [comment]
Added Klass via [ceed8f0](https://github.com/osmlab/name-suggestion-index/commit/ceed8f08d44f52da335c1c6b0b4eb44305f81ed3)

**Comment by cr33dx** [comment]
@UKChris-osm where should body care data be at ?

**Comment by UKChris-osm** [comment]
I would suggest **Bodycare** is better suited to "_shop/cosmetics_", as it seems to be a similar type of shop to **The Body Shop** which is already in that same category.

**Comment by lakedistrictOSM** [comment]
I'd disagree. Their shops tend to sell a wide range of toiletries (shampoo, toothpaste, first aid items, medicines, make up, healthcare products etc) rather than just or primarily cosmetics. The wiki definition for shop=chemist https://wiki.openstreetmap.org/wiki/Tag:shop%3Dchemist "A shop selling articles of personal hygiene, cosmetics, and household cleaning products." seems the best fit.

The wiki definition for shop=cosmetics is "Shop exclusively selling cosmetics" https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcosmetics

**Comment by cr33dx** [comment]
@UKChris-osm @lakedistrictOSM so cosmetics or chemist?

**Comment by bhousel** [comment]
It sounds like the UK folks have settled on `shop=chemist` , and I agree if Bodycare also sells things like toothpaste, first aid, medicine, etc.

It's worth mentioning that we already match group these tags because they are so similar:
```js
    "convenience": [
      "shop/beauty",
      "shop/chemist",
      "shop/convenience",
      "shop/cosmetics",
      "shop/newsagent"
    ],
```

Meaning - if someone did tag a Bodycare as `shop=convenience` or `shop=cosmetics`, the matcher would still match the entry and suggest that the user switch it to the `shop=chemist` tag.

**Comment by camilorecce** [comment]
Added Bodycare: 99c5e49ca423f16805008ab6fd73d3230bfd9889

**Comment by lakedistrictOSM** [comment]
Thanks @UKChris-osm, @cr33dx and @camilorecce for adding these 😃

**Comment by UKChris-osm** [comment]
Bodycare merged via #4344 

I initially went for "_cosmetics_" as that's all they seem to list on their web site :)
